common/templates/general.go

```diff
@@ -240,6 +241,29 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 				ContentType: "text/plain",
 				Reader:      &buf,
 			}
+
+		case "image":
+			if val == nil {
+				return nil, errors.New("need an argument for the image")
+			}
+			str := fmt.Sprint(val)
+			img, err := http.Get(str)
```

Comment:
You can perhaps check if `str != ""`? 
http.Get would of course handle that and error, but I think this is a better approach. Totally subjective though.